hri_c_driver/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(hri_c_driver)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
```

Comment:
This doesn't actually depend on roscpp, correct?  If not, let's just remove that dependency here (and in the package.xml below).

Reply:
Removed in 5d0a284